### PR TITLE
USD : Build usdImaging

### PR DESCRIPTION
**Pull request by johnhaddon**
This provides `usdview` etc. It also requires OpenSubdiv which has been added as well.

### CODE REVIEW

File: OpenSubdiv/config.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	"license" : "LICENSE.txt",
+
+	"dependencies" : [ "TBB" ],
+
+	"commands" : [
+
+		"cmake"
+			" -D CMAKE_INSTALL_PREFIX={buildDir}"
+			" -D CMAKE_PREFIX_PATH={buildDir}"
+			" -D NO_GLFW=1"
```
Comment (themissingcow) [inline OpenSubdiv/config.py:20]: We may want ` -D NO_CUDA=1` here or ` -D OSD_CUDA_NVCC_FLAGS='--gpu-architecture compute_50'` or similar, as the defaults don't build on anything vaguely modern.

**Review by themissingcow — CHANGES_REQUESTED** [review]
Hey John, be good to get this in, but it's highlighted a few missing deps in a clean build.

File: OpenSubdiv/config.py (hunk 1 above)
Comment (themissingcow) [reply OpenSubdiv/config.py:20]: https://github.com/PixarAnimationStudios/OpenSubdiv/issues/965

File: USD/config.py [hunk 2]
```diff
@@ -10,11 +10,12 @@
 
 	"license" : "LICENSE.txt",
 
-	"dependencies" : [ "Boost", "Python", "OpenImageIO", "TBB", "Alembic" ],
+	"dependencies" : [ "Boost", "Python", "OpenImageIO", "TBB", "Alembic", "OpenSubdiv", "PyOpenGL" ],
```
[the remark this answers is not in the document]
Comment (johnhaddon) [reply USD/config.py]: Thanks for catching that Tom. I had done the lazy thing and just tested a build on top of an existing complete build. I've started from scratch now and fixed this problem and a few others that cropped up. This fix is squashed into 
1777079.

File: OpenSubdiv/config.py (hunk 1 above)
Comment (johnhaddon) [reply OpenSubdiv/config.py:23]: Thanks - also fixed in 1777079. I've actually disabled a whole load more after discovering that they are disabled in the builds made by https://github.com/PixarAnimationStudios/USD/blob/release/build_scripts/build_usd.py. Note also one new commit, 99cca5e, which fixes a problem running `pyside2-uic` without having built Subprocess32 first.

**Review by themissingcow — APPROVED** [review]
Thanks John, this seemed to work fine for me from a completely fresh build.

**Comment by themissingcow** [comment]
@johnhaddon Sorry I just unintentionally sabotaged you by merging that Changes PR.

**Pull request merged**